quicklogic/common/toolchain_wrappers/symbiflow_place

```diff
@@ -45,6 +45,12 @@ else
 
 fi
 
+# Generate lib file
+if [ -s $PART ]; then
```

Comment:
Hmmm, Why this is generated during placement? Shouldn't there be a separate option in `ql_symbiflow` to do it?

Reply:
I didn't see an option to generate these lib file in `ql_symbiflow`.

@kkumar23, do you know whether this should be here or is it better to move it in `ql_symbiflow`?

Reply:
Given that the lib data files are being removed and will be added to the external data repository, this change is not part of this PR anymore. As soon as the plugin will contain `lib` and `csv` files the lib data generation will be implemented once again.